packages/katello/rubygem-qpid_proton/rubygem-qpid_proton.spec

```diff
@@ -7,8 +7,8 @@
 %global qpid_version 0.33.0
```

Comment:
This is the line that needs updating so that the dependencies get the right version

Reply:
oh thanks!